src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -279,6 +284,11 @@ private static void AddBracketIndentationOperation(List<IndentBlockOperation> li
         }
     }
 
+    private static bool IsSwitchExpressionPattern(SyntaxNode node)
```

Comment:
nit: i would *personally* inline this.  It's only called in one place, and has very simple logic.  If you do want to keep this as a helper method, that's ok. 

--

"nits" are ideas/gentle-suggestions.  But i'm deferring to your judgement here on what you prefer.